src/plots/attributes.js

```diff
@@ -129,6 +129,19 @@ module.exports = {
         ].join(' ')
     },
     hoverlabel: fxAttrs.hoverlabel,
+    hovertemplate: {
+        valType: 'string',
+        role: 'info',
+        dflt: false,
+        editType: 'none',
+        description: [
```

Comment:
It might be nice to add support for `arrayOk:true` `hovertemplate`, e.g.

```js
trace = {
  x: [1, 2, 3],
  y: [2, 1, 2],
  hovertemplate: ['first pt<br>%{y}', '%second pt<br>%{y}<extra>LOOK!</extra>', '']
}
```

Reply:
... it be _should_ be fairly to do so:

- handle `hovertemplate` in https://github.com/plotly/plotly.js/blob/master/src/components/fx/calc.js
- add some logic for `calcdata[i][j].ht` (`ht` for `hovertemplate`) in `fx/hover.js`
- do something similar in `pie/plot.js` using `helpers.castOption()`